### PR TITLE
Multix-a-simple-UI-for-complex-multisigs_1.md

### PR DESCRIPTION
**Pull request by Tbaut**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the DAI payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/1451

### CODE REVIEW

**Comment by 0xCaso** [comment]
Hi @Tbaut, thanks for the delivery! I'll start the evaluation soon, and I'll get back to you asap.

File: deliveries/Multix-a-simple-UI-for-complex-multisigs_1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+We have delivered the interface, it's live on Rococo and it can be found here: https://multix.chainsafe.io/
+There are a couple UX improvements to be made in the very short term, notably for the multisig creation flow, those can be found [in the issue tracker](https://github.com/ChainSafe/Multix/issues).
+
+**Deliverables**
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0.b | Documentation | [Multix repo](https://github.com/ChainSafe/Multix) | The code is commented in line and the flows should be self explanatory for users. | 
+| 0.d  | Docker | [Multix indexer](https://github.com/ChainSafe/Multix/tree/main/squid) | The docker of the indexer is available in the repo | 
+| 0.e  | Article | [Medium article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d) | An article explaining what Multix brings and educating about proxies has been published|
```
Comment (0xCaso) [inline deliveries/Multix-a-simple-UI-for-complex-multisigs_1.md:18]: Add License delivery (and adjust format)
```suggestion
| ------------- | ------------- | ------------- |------------- |
| 0a. | License | [LICENSE](https://github.com/ChainSafe/Multix/blob/main/LICENSE) | Apache 2.0 | 
| 0b. | Documentation | [Multix repo](https://github.com/ChainSafe/Multix) | The code is commented in line and the flows should be self explanatory for users. | 
| 0d.  | Docker | [Multix indexer](https://github.com/ChainSafe/Multix/tree/main/squid) | The docker of the indexer is available in the repo | 
| 0e.  | Article | [Medium article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d) | An article explaining what Multix brings and educating about proxies has been published| 
```

**Review by 0xCaso — COMMENTED** [review]
Could you make this minor change please?

File: deliveries/Multix-a-simple-UI-for-complex-multisigs_1.md (hunk 1 above)
Comment (Tbaut) [reply deliveries/Multix-a-simple-UI-for-complex-multisigs_1.md]: of course, thanks for suggesting it, makes things much simpler :)

**Comment by 0xCaso** [comment]
May you add a super-quick "Docker Setup" guide [here](https://github.com/ChainSafe/Multix/blob/main/squid/README.md) please?

**Comment by 0xCaso** [comment]
Thank you! I was meaning a mini-guide that explains how to launch the indexer through the `Dockerfile` and `docker-compose.yml` you provided in the repo (like: run `docker-compose up` to launch this, build the `Dockerfile`...)

**Comment by Tbaut** [comment]
I added some more info, although it's a little redundant with the part above using `make`. https://github.com/ChainSafe/Multix/pull/17

**Review by 0xCaso — APPROVED** [review]
I'm gonna approve it, good job!

(We had a conversation on Element, where he helped me for the setup and we found a minor issue in the `.env.example` file)

**Review by Noc2 — APPROVED** [review]

**Comment by 0xCaso** [comment]
@Tbaut I ask you one last thing, could you resend the invoice specifying the VAT Amount please (also if it's 0)?

**Comment by 0xCaso** [comment]
Hi @Tbaut I've just forwarded internally the invoice

**Comment by RouvenP** [comment]
hi @Tbaut we just transferred the payment

**Pull request merged**